Docs/source/running_cpp/parameters.rst

```diff
@@ -621,6 +621,7 @@ Numerics and algorithms
 
      - ``boris``: Boris pusher.
      - ``vay``: Vay pusher (see `Vay, Phys. Plasmas (2008) <https://aip.scitation.org/doi/10.1063/1.2837054>`__)
+     - ``hc``: Higuera-Cary pusher (see `Higuera and Cary, Phys. Plasmas (2017) <https://aip.scitation.org/doi/10.1063/1.4979989>`__)
```

Comment:
Thanks for adding the doc!
Just a suggestion: could we use a more explicit name, like `higuera` instead of `hc`?